docs/features/versions.mdx

```diff
@@ -29,15 +29,44 @@ After your build completes, Zephyr uploads only new or changed files to its glob
 
 ### Version Access and Persistence
 
-Every version gets a permanent URL that works forever. Zephyr creates a unique version identifier by combining your username, build ID, and application name. This means each build gets its own distinct version, even if you're building the same code in different situations.
+Every version gets a permanent URL that works forever. Each version is uniquely identified by a content-based `snapshot_id` (SHA-256 hash) that represents the exact state of your build. Zephyr also assigns a semantic version number that automatically increments with each build, and generates a unique URL hostname for accessing your version.
 
-When Zephyr creates a version, it takes a complete snapshot of your deployment. This snapshot includes everything: who built it, when it was built, which git branch and commit it came from, what dependencies it uses, and a detailed list of all files with their fingerprints. This gives you complete visibility into every version's history.
+**Version URL Format:**
+
+```
+https://{username}-{buildId}-{app-name}-{project}-{hash}.{domain}
+```
+
+**Example:** `https://john-doe-42-header-component-ecommerce-abc123.zephyr-cloud.io`
```

Comment:
Should we add `*-ze` or `*.ze` as the default suffix here?
we do have it and the separator is Cloud specific